File: MethodOR.java
class Base {
    void print() {
        System.out.println("This is a base class");
    }
}

class Derived extends Base {
    void print() {
        System.out.println("This is a Derived class");
    }
}

public class MethodOR {
    public static void main(String[] args) {
        Derived obj = new Derived();
        obj.print();
    }

}

File: MethodOR.cpp
#include <iostream>
using namespace std;
class Base
{
public:
    void print()
    {
        cout << "This is a base class" << endl;
    }
};
class Derived : public Base
{
public:
    void print()
    {
        cout << "This is a Derived class" << endl;
    }
};
int main()
{
    Derived obj;
    obj.print();
    return 0;
}